WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackRedirector.swift

```diff
@@ -8,15 +9,7 @@ class JetpackRedirector {
     /// Note: The string values should kept in-sync with Jetpack's URL scheme.
     ///
     static var jetpackDeepLinkScheme: String {
-        /// Important: Multiple compiler flags are set for some builds
-        /// so ordering matters.
-        #if DEBUG
-        return "jpdebug"
-        #elseif ALPHA_BUILD
-        return "jpalpha"
-        #else
-        return "jetpack"
-        #endif
+        BuildSettings.current.jetpackAppURLScheme
```

Comment:
Coming here without much context, does this mean there will be a unique deep link for `jetpack`? Is this what we want? I think it makes sense, but I'm not sure why there were different deep link schemes before.

Reply:
There are no functional changes. `BuildSettings.current.jetpackAppURLScheme` returns the same values depending on your build configuration. We currently have three: debug, alpha, and release. Preproduction builds do use separate URL schemes for the app.

Reply:
Can we just use `appURLScheme`, which should return the correct value based on the build configuration and running app?

Reply:
@crazytonyli 

> Can we just use appURLScheme, which should return the correct value based on the build configuration and running app?

I think we _need_ a dedicate scheme for Jetpack because that's the one we use to open the Jetpack app from the WordPress app?

Reply:
Oh right. That makes sense 👍